project-base/app/src/DataFixtures/Demo/DataObjects/ParameterDataFixtureData.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\DataFixtures\Demo\DataObjects;
```

Comment:
A namespace is usually named in singular, can you please rename it to `DataObject`?

Reply:
Sure. Changed it to singular form.